README.md

```diff
@@ -45,6 +45,54 @@ Yes! This work is still in early development, which means it's _not ready for pr
 
  [![asciicast](https://asciinema.org/a/471709.svg)](https://asciinema.org/a/471709)
 
+## Workspaces in v0.5.0
+
+The workspaces are arranged in a tree.  Each vertex in the tree has a name.  The name of the root vertex is `root`, and no other vetex may have that name.  Other vertices can be addressed by a pathname formed by putting all the path's vertex names together, separated by colons.  For example, the pathname `root:default:foo` identifies the child `foo` of the child `default` of the `root`.
+
+The `kubectl kcp workspace` CLI can be used to work with workspaces.  You can use `ws` as a shorthand for `workspace`.
+
+There are four kinds of workspaces, as follows.
```

Comment:
... by default in KCP. Any service built on-top of KCP can use different hierarchies. These are just means as examples.

Reply:
Revised.